lib/websocket/handshake/base.rb

```diff
@@ -3,6 +3,8 @@ module Handshake
     # @abstract Subclass and override to implement custom handshakes
     class Base
       include ExceptionHandler
+      # Recreate inspect as #to_s will be overwritten
```

Comment:
Sure! Although I'm not sure if it is true. As I can see the `#to_s` method is not overwritten in the second case of the inclusion.